server/src/main/java/org/elasticsearch/transport/RemoteConnectionManager.java

```diff
@@ -83,7 +83,14 @@ public void removeListener(TransportConnectionListener listener) {
 
     @Override
     public void openConnection(DiscoveryNode node, ConnectionProfile profile, ActionListener<Transport.Connection> listener) {
-        delegate.openConnection(node, profile, listener);
+        delegate.openConnection(
```

Comment:
Change from this PR: https://github.com/elastic/elasticsearch/pull/92584 -- pending approval from distributed team.